packages/flutter/lib/src/material/chip.dart

```diff
@@ -344,9 +1579,11 @@ class _ChipRenderTheme {
     return typedOther.avatar == avatar &&
         typedOther.label == label &&
         typedOther.deleteIcon == deleteIcon &&
-        typedOther.container == container &&
         typedOther.padding == padding &&
-        typedOther.labelPadding == labelPadding;
+        typedOther.labelPadding == labelPadding &&
```

Comment:
The latest style for this sort of thing puts the `&&` on the left so that all of the clauses line up:
```
    return typedOther.foo == foo
        && typedOther.bar == bar
        && typedOther.baz == baz

Reply:
Just to be clear, I'll make these changes before I submit, but I don't want to do pure formatting changes that don't match what dartfmt does until I'm done with all the other changes.